portal-kernel/src/com/liferay/portal/kernel/search/background/task/ReindexBackgroundTaskConstants.java

```diff
@@ -20,6 +20,10 @@ public class ReindexBackgroundTaskConstants {
 
 	public static final String EXECUTION_MODE = "executionMode";
 
+	public static final String INDEX_REINDEXER_END = "indexReindexerEnd";
+
+	public static final String INDEX_REINDEXER_START = "indexReindexerStart";
```

Comment:
it looks like PORTAL_START and SINGLE_START are used in ReindexBackgroundTaskStatusMessageTranslator to help calculate the progress percentage... im wondering if we should reuse one of those instead, or if we need to create unique logic for INDEX_REINDEXER_START when calculating percentage?

Reply:
Yeah it makes sense now why the percentage was not showing 😆 

Realise it today and was trying to figure out what to do